src/main/scala/temple/generate/docker/DockerFileGenerator.scala

```diff
@@ -0,0 +1,30 @@
+package temple.generate.docker
+
+import io.circe.Printer
+import io.circe.syntax._
+import temple.generate.docker.ast.Statement._
+import temple.generate.docker.ast.{DockerfileRoot, Statement}
+import temple.generate.utils.CodeTerm.mkCode
+
+/** Generator object for building Dockerfiles from the Dockerfile AST objects */
+object DockerFileGenerator {
```

Comment:
Just a consistency one, think we should go with `Dockerfile` rather than `DockerFile`